server/api/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+
+const apiRouter = express.Router();
+
+//TODO: Fix issue that is making localhost:3000/api return with error saying "Cannot GET /api"
```

Comment:
I'm not encountering this issue anymore?

Reply:
We thought we deleted that TODO